tests/unit/test_install.py

```diff
@@ -0,0 +1,134 @@
+import os
+from unittest.mock import create_autospec
+
+import pytest
+from databricks.labs.blueprint.installation import MockInstallation
+from databricks.labs.blueprint.tui import MockPrompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.install import WorkspaceInstaller
+
+
+@pytest.fixture
+def ws():
+    ws = create_autospec(WorkspaceClient)
+
+    ws.product = "remorph"
+    ws.product_version = __version__
+
+    return ws
+
+
+@pytest.fixture
+def mock_installation():
+    return MockInstallation(
+        {"state.json": {"version": 1, "source": "snowflake", "skip_validation": True, "catalog_name": "remorph_test"}}
+    )
+
+
+def test_install(ws, mock_installation):
+    prompts = MockPrompts(
+        {
+            r"Select the source": "0",
+            r"Do you want to Skip Validation": "1",
+            r"Enter catalog_name": "remorph_test",
+            r"Enter schema_name": "remorph_schema",
+            r".*": "",
+        }
+    )
+    install = WorkspaceInstaller(prompts, mock_installation, ws)
+    install.run()
+
+
+def test_install_dbr(ws, mock_installation):
+    os.environ["DATABRICKS_RUNTIME_VERSION"] = "14.1"
```

Comment:
i think you should do the monkey patch for this one, otherwise tests will be flaky

Reply:
Done